homeassistant/components/esphome/dashboard.py

```diff
@@ -83,6 +84,19 @@ def __init__(
         self.url = url
         self.api = ESPHomeDashboardAPI(url, session)
 
+    @property
+    def supports_update(self) -> bool:
+        """Return whether the dashboard supports updates."""
+        if self.data is None:
+            raise RuntimeError("Data needs to be loaded first")
+
+        if len(self.data) == 0:
+            return False
+
+        esphome_version: str = next(iter(self.data.values()))["current_version"]
+
+        return AwesomeVersion(esphome_version) >= AwesomeVersion("2023.2.0-dev")
```

Comment:
I dont think the `-dev` is needed here
```suggestion
        return AwesomeVersion(esphome_version) >= AwesomeVersion("2023.2.0")
```

Reply:
Nevermind I was wrong. But perhaps:
```suggestion
        return AwesomeVersion(esphome_version) > AwesomeVersion("2023.1.0")
```

Reply:
but that would match 2023.1.1 ?

Reply:
```suggestion
        # There is no January release
        return AwesomeVersion(esphome_version) > AwesomeVersion("2023.1.0")
```